src/components/ColorizedMenuItem.tsx

```diff
@@ -17,7 +15,7 @@ export const ColorizedMenuItem: FC<{
     const hoverColor = useMemo(() => colorLayer(hue, 5, hue === 1 ? 0 : undefined)[themeId], [hue, themeId]);
 
     return (
-        <MarkedListItem hoverColor={hoverColor} mark={<StateDot hue={hue} />} {...props}>
+        <MarkedListItem hoverColor={hoverColor} mark={<></>} {...props}>
```

Comment:
It affects all dropdown instances. Ex: StateFilter. We need to change Priority only.